scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
Many configure scripts test the contents of $CC, gcc, and cc, in that order. Some honour CROSS_COMPILE= for a prefix. It looks like ct-ng was setting CROSS_COMPILE, but GMP ignored it.

You are correct that I didn't run ct-ng build-all. I can this evening... with this change.